File: spring-springmvc-springdatajpa/src/main/java/com/jiangKlijna/web/dao/UserDao.java
package com.jiangKlijna.web.dao;

import com.jiangKlijna.web.bean.User;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

/**
 * Created by leil7 on 2017/6/19. spring-springmvc-springdatajpa
 */
public interface UserDao extends JpaRepository<User, Integer> {

}

File: springmvc/spring-springmvc-springdatajpa-kotlin/src/main/java/com/jiangKlijna/web/dao/UserDao.kt
package com.jiangKlijna.web.dao

import com.jiangKlijna.web.bean.User
import org.springframework.data.jpa.repository.JpaRepository

/**
 * Created by leil7 on 2017/6/19. spring-springmvc-springdatajpa
 */
interface UserDao : JpaRepository<User, Int>
